concepts/observability-strategy/strategy.md

```diff
@@ -1,54 +1,60 @@
-- Monitoring is based on Prometheus, which is widely adopted but limited to the Prometheus protocols. Users can configure the monitoring stack only during the Kyma upgrade process.
+## Current Situation 
 
-Integration (and with that, changing the focus away from in-cluster backends) is the key to open up the stack for a broad range of use cases. Users can bring their own backends if they already use a commercial offering or run their own infrastructure. The data can be stored outside the cluster in a managed offering, shared with the data of multiple clusters, away from any tampering or deletion attempt of a hacker, to name just a few.
+Observability can be split up in the following stages:
+1. Instrumentation of the user application
+2. Collection and preprocessing of the signals from the user application and the surrounding infrastructure, including metadata enrichment
+3. Delivery of the signal to an backend for storage
+4. Optionally, aggregation of the signals
+5. Storage of the signals
+6. Analysis, querying, and dashboarding of the signals
 
-Providing ready-to-use in-cluster backends is necessarily opinionated, does not cover all usage scenarios, and does not fit into Kyma's goal of providing the Kubernetes building blocks to integrate into the the SAP ecosystem. Also, the licensing issues (particularly with Grafana and Loki, or ElasticSearch as an alternative backend technology) show that an opinionated stack is problematic. It's better to handle opinionation by integrating with actual managed services. 
+![a](./assets/strategy-stages.drawio.svg)
 
-The following strategy proposes a new approach for the Kyma observability stack by opening up to new scenarios by supporting convenient integration at runtime. At the same time, it reduces the focus on ready-to-use solutions running inside the cluster.
+The current Kyma observability stack is spreading over all stages and is providing a lightweight end-to-end setup for basic needs. Hereby, configuration possibilities are available at installation time mainly.
 
-## Goals
+![Current observability stack](./assets/strategy-current.drawio.svg "Current observability stack")
 
-### Stages of observability
+- Application Logs
+  - Instrumentation is done by printing logs to stdout/stderr as recommended by Kubernetes best practices, system components are following that guide and logs are available already; no other way of getting logs collected
+  - FluentBit as log collector defines a hardcoded pipeline for collecting, enriching and pushing the logs to a backend; no further configuration of pipelines at runtime, especially additional outputs for external systems cannot be configured
+  - Storage is realized by a lightweight (non-scalable) Loki installation, no configuration at runtime
+  - Reporting is implemented by the log explorer of Grafana
+- Metrics
```

Comment:
Consider formatting this as h3 and reducing the indentation for the sub-bullets.

```suggestion
### Metrics
```